PMSuperButtonSample/Assets.xcassets/glyphicons-basic-790-fish.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
could you please use `pdf` instead of `png` slices? Then u'll be able to use one source for that